mep3_simulation/webots_data/controllers/opponent_robot/opponent_robot.py

```diff
@@ -121,14 +116,14 @@ def main():
                 delta_x = math.cos(target_angle) * velocity_factor
                 delta_y = math.sin(target_angle) * velocity_factor
 
-                if not are_colliding(box_big_translation,
+                if not are_colliding(opponent_field,
                                      memristor_robot_translation
                                      ) and supervisor.getName() == 'opponent_box_big':
```

Comment:
Zasto ovdje provjeravas `and supervisor.getName() == 'opponent_box_big'`?

Reply:
Ovo mi je ostalo od ranije.